godot-core/src/meta/mod.rs

```diff
@@ -54,10 +54,12 @@ mod signature;
 mod traits;
 
 pub mod error;
+pub mod property_update;
 
 pub use args::*;
 pub use class_name::ClassName;
 pub use godot_convert::{FromGodot, GodotConvert, ToGodot};
+pub use property_update::PropertyUpdate;
```

Comment:
Should not be a `pub` module, just re-export it publicly.

I think [`register::property`](https://godot-rust.github.io/docs/gdext/pr-946/godot/register/property/index.html) would fit better as a module though, could you move these imports + `property_update.rs`?